OpenCL_C.txt

```diff
@@ -8827,13 +9112,13 @@ events.
 | |
 | void *capture_event_profiling_info*(clk_event_t _event_,
   clk_profiling_info _name_, global void *_value_)
-    | Captures the profiling information for functions that are enqueued as
+   a| Captures the profiling information for functions that are enqueued as
```

Comment:
Minor: With this change the indenting below for `CLK_PROFILING_COMMAND_EXEC_TYPE` is unnecessary and should be removed.

Reply:
Fixed.